### DISCUSSION TITLE
Is there a way to display PDFs larger than 200 inches?

### DISCUSSION THREAD

**Discussion by usagibear** [wojtekmaj/react-pdf#1373] [Q&A]
I have an app that converts some files to PDF. When I open the file in Chrome or Edge, it loads, albeit slowly. In Abode Acrobat Reader, it loads the file but truncated. The reason Acrobat truncates the PDF is likely because of the limit of 200 inches on either side of a page. 

However, the file does not load correctly since that the pages are blank in with a React-PDF viewer, likely due to the same limitation. Is there anything I can do about this inside of React-PDF?

The errors are as follows:

```
PDF.js v3.4.62 (build: 9cea76483)
Message: 
Stack: putBinaryImageData@resource://pdf.js/build/pdf.js:5135:11
paintInlineImageXObject@resource://pdf.js/build/pdf.js:6703:25
paintImageXObject@resource://pdf.js/build/pdf.js:6663:10
executeOperatorList@resource://pdf.js/build/pdf.js:5475:20
_next@resource://pdf.js/build/pdf.js:2614:37
__webpack_modules__</_scheduleNext/<@resource://pdf.js/build/pdf.js:2604:14
```

```
renderView: "[Exception... "Failure"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://pdf.js/build/pdf.js :: putBinaryImageData :: line 5135"  data: no]"
```

**Comment by wojtekmaj** [comment]
Can't tell for sure, especially without reproducible example, but you may experiment with maxImageSize passed to options prop. It looks from the error message like it crashes on the image so this could, at least in theory, prevent the crash.
